crds/v1alpha1/src/main/crds/applicationsets.argoproj.io

```diff
@@ -15295,7 +15295,7 @@ spec:
               applicationStatus:
                 items:
                   properties:
-                    application:
+                    applicationConfig:
```

Comment:
This definitely doesn't seem right. I don't think that CRDs changed since last time I pulled them.